ui/job-view/details/tabs/TabsPanel.jsx

```diff
@@ -140,7 +140,7 @@ class TabsPanel extends React.Component {
     const countPinnedJobs = Object.keys(pinnedJobs).length;
 
     return (
-      <div id="tabs-panel">
+      <div id="tabs-panel" role="region" aria-label="Tabs Container">
```

Comment:
I don't feel that "Tabs Container" is necessarily clearer and it's more verbose. What about just "Job", since this is all content about the particular job selected? (Screen readers will generally report that as "Job region".) If that seems off to you, let's just go with "Tabs" as you originally suggested.

Reply:
Sure, "Job" works.  I'll go with that.  We can always adjust later, if need be.